core/src/main/java/org/bitcoinj/coinjoin/CoinJoinBroadcastTx.java

```diff
@@ -70,6 +72,18 @@ public CoinJoinBroadcastTx(
         this.signatureTime = signatureTime;
     }
 
+    public CoinJoinBroadcastTx(
+            NetworkParameters params,
+            Transaction tx,
+            TransactionOutPoint masternodeOutpoint,
+            long signatureTime
+    ) {
+        super(params);
+        this.tx = tx;
+        this.masternodeOutpoint = masternodeOutpoint;
+        this.signatureTime = signatureTime;
+    }
```

Comment:
More constructors like this were added to support the `CoinJoinServer` class.  Many of these classes would never be created by a client except when deserializing a received message.